lib/active_record_shards/model.rb

```diff
@@ -50,11 +50,28 @@ def from_shard
 
     def self.extended(base)
       base.send(:include, InstanceMethods)
+      base.singleton_class.send(:alias_method, :primary_key_without_default_value, :primary_key)
```

Comment:
`@primary_key` annoyingly is a class level instance variable, even if we set ActiveRecord::Base.primary_key, it doesn't get inherited. Couldn't think of a better way of doing this, lemme know if anyone has suggestions